src/universal/modules/welcome/components/WelcomeWizardForms/index.js

```diff
@@ -0,0 +1,3 @@
+export Step1PreferredName from './Step1PreferredName';
```

Comment:
Personal preference, I like names based solely on content (think URN or a magnet link) vs names based on location (think URL). Reasoning is that `preferredName` won't always live in `step1`, or another step might mutate it so it ends up being shared by many steps, etc. But if you like it like that, I can dig it.